docs/src/pages/components/loader/react.mdx

```diff
@@ -12,99 +21,96 @@ import { ComponentStyleDisplay } from '@/components/ComponentStyleDisplay';
 
 Import the Loader component and styles.
 
-```jsx
-import { Loader } from '@aws-amplify/ui-react';
-import '@aws-amplify/ui-react/styles.css';
+<Example>
+  <DefaultLoaderExample />
+  <ExampleCode>
+    ```jsx file=./examples/DefaultLoaderExample.tsx
 
-<Loader />;
-```
+    ```
 
-<Example>
-  <Loader />
+  </ExampleCode>
 </Example>
 
 ### Sizes
 
 Use the `size` prop to change the Loader size. Available options are `small`, `large`, and none (default).
 
-```jsx
-import { Loader } from '@aws-amplify/ui-react';
-import '@aws-amplify/ui-react/styles.css';
+<Example>
+  <LoaderSizesExample />
+  <ExampleCode>
+    ```jsx file=./examples/LoaderSizesExample.tsx
 
-<Loader size="small"/>
-<Loader />
-<Loader size="large"/>
-```
+    ```
 
-<Example>
-  <Flex alignItems="center">
-    <Loader size="small" />
-    <Loader />
-    <Loader size="large" />
-  </Flex>
+  </ExampleCode>
 </Example>
 
 ### Variation
 
 Loader comes in 2 variants, `linear` and none(default). Pass the `variation` prop and set it to either of these values.
 
-```jsx
-import { Loader } from '@aws-amplify/ui-react';
-import '@aws-amplify/ui-react/styles.css';
+<Example>
+  <LoaderVariationExample />
+  <ExampleCode>
+    ```jsx file=./examples/LoaderVariationExample.tsx
 
-<Loader />
-<Loader variation="linear" />
-```
+    ```
 
-<Example>
-  <Loader />
-  <Loader variation="linear" />
+  </ExampleCode>
 </Example>
 
 ### Colors
 
 Pass `filledColor` and `emptyColor` props to color your own Loader.
 
-```jsx
-import { Loader } from '@aws-amplify/ui-react';
-import '@aws-amplify/ui-react/styles.css';
-
-  <Loader
-    emptyColor="var(--amplify-colors-black)"
-    filledColor="var(--amplify-colors-orange-40)"
-  />
-  <Loader
-    variation="linear"
-    emptyColor="var(--amplify-colors-black)"
-    filledColor="var(--amplify-colors-orange-40)"
-  />
-```
+<Example>
+  <LoaderColorExample />
+  <ExampleCode>
+    ```jsx file=./examples/LoaderColorExample.tsx
+
+    ```
+
+  </ExampleCode>
+</Example>
+
+### Determinate
+
+To use a determinate loader, set `isDeterminate` to `true` and pass `percentage`.
```

Comment:
Should we add a short description what determinate loaders are? Personally I wasn't sure what determinate loaders are before I saw your gif.

Reply:
That said, the `<DeterminateLoaderExample />` may be sufficient on docs, up to you!

Reply:
Thanks for the feedback! Let me add one.